pimd/pim_rp.h

```diff
@@ -75,11 +76,11 @@ int pim_rp_set_upstream_addr(struct pim_instance *pim, pim_addr *up,
 
 struct pim_rpf *pim_rp_g(struct pim_instance *pim, pim_addr group);
 
-#define I_am_RP(P, G)  pim_rp_i_am_rp ((P), (G))
```

Comment:
unrelated whitespace change, please remove it
(even if clang-format complains — should not do a whitespace change mixed between other changes)